src/Compilers/VisualBasic/Portable/Parser/BlockContexts/CompilationUnitContext.vb

```diff
@@ -115,7 +117,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
                                                        declarations,
                                                        optionalTerminator)
 
-            Dim regionsAreAllowedEverywhere = Not haveRegionDirectives OrElse Parser.CheckFeatureAvailability(Feature.RegionsEverywhere)
+            Dim regionsAreAllowedEverywhere = Not haveRegionDirectives OrElse Feature.RegionsEverywhere.IsAvailable(_parser._scanner.Options)
```

Comment:
`Parser` could expose an `Options` property rather than exposing `_scanner` which is arguably an implementation detail.